src/hooks/useCart/useCart.test.tsx

```diff
@@ -159,6 +164,13 @@ const mockPostTransactionResult = {
   ]
 };
 
+const mockQuotaNotEligible: any = (id: string) => {
```

Comment:
Small nit too: might be more understandable if the quota word was replaced here, cuz it's not really that the quota that was not eligible

Reply:
got it, changed! :)